### ISSUE TITLE
Get Image with names of project, dataset and image

### ISSUE THREAD

**Issue by mcib3d** [GReD-Clermont/simple-omero-client#24]
Hi,

Usually I store my images as a set of 3 names : its project name, its dataset name and its image name.

Would it be possible to get a function for that, I guess directly in client ? 

Thanks.

**Comment by ppouchin** [comment]
Hi!

I'm not sure I understand... 
You would like to rename images so that their names also contain the dataset name and the project name?

**Comment by mcib3d** [comment]
Hi,

Sorry it was not very clear, ideally I would like something like this:
`ImageWrapper image = client.getImage(projectName, datasetName, imageName`
Thanks

**Comment by ppouchin** [comment]
Ok, I see.
However, technically, nothing guarantees the unicity of the (project, dataset, image) association.  
For each level, several objects could share the same name.
The best that could be done would be something like:

```java
public List<ImageWrapper> getImages(String projectName, String datasetName, String imageName)
throws ServiceException, AccessException, ExecutionException {
    List<ImageWrapper> images = new ArrayList<>();
    List<ProjectWrapper> projects = getProjects(projectName);
    for(ProjectWrapper project : projects) {
        List<DatasetWrapper> datasets = project.getDatasets(datasetName);
        for(DatasetWrapper dataset : datasets) {
            images.addAll(dataset.getImages(this, imageName));
        }
    }
    return images;
}
```

**Comment by ppouchin** [comment]
Normally, it should be ok. By default, the methods return empty lists if nothing is found.

I just need to write tests for both issues now. This one should be quick (but the import & replace requires a more complex setup).

**Comment by mcib3d** [comment]
Happy to test :)

**Comment by ppouchin** [comment]
Solved with the release of [5.9.1](https://github.com/GReD-Clermont/simple-omero-client/releases/tag/5.9.1).
